### PR TITLE
Ssh keys

### PR DESCRIPTION
**Pull request by lindseybateman**

### CODE REVIEW

**Comment by lindseybateman** [comment]
i could use some help making the content a bit richer or tips on what i could add to this section

File: content/docs/ssh-keys/using-ssh-keys.md [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+
+
+Before you can actucally use a `SSH Key pair` you will have to create an
+environment variable that will make use of a `SSH Key pair`.
+
+[Read more on creating environment variables &rsaquo;](/docs/environment-variables/creating-env-vars.html)
+
+### Add to known hosts pipeline step
+
+Using this step in combination with a `SSH Key pair` makes it possible to deploy your code.
```
Comment (flenter) [inline content/docs/ssh-keys/using-ssh-keys.md]: Maybe change this into something like: 

> Depending on the box you are using, you may want to use this step so builds/deploys don't halt when you setup a connection with them:
> ```
> The authenticity of host 'some-server.wercker.com (1.2.3.4)' can't be established.
> RSA key fingerprint is ff:ee:dd:cc:bb:aa:99:88:77:66:55:44:33:22:11:00.
> Are you sure you want to continue connecting (yes/no)?
> ```

File: content/docs/ssh-keys/using-ssh-keys.md [hunk 2]
```diff
@@ -1,5 +1,37 @@
...
+    - add-to-known_hosts:
+        hostname: ssh.example.com
+        fingerprint: ce:83:e9:7d:02:a4:e3:63:3f:8a:07:cc:d5:d9:bb:cd
+```
+
+[Read more on the steo add-to-kwown_hosts &rsaquo;](https://app.wercker.com/#applications/521764dde36a64ff110022f2/tab/details)
+
+### Add ssh key pipeline step
+
+This step can be used to write these values to an `EdentityFile` and add them to the SSH configuration.
```
Comment (flenter) [inline content/docs/ssh-keys/using-ssh-keys.md]: typo: `identity file`?

File: content/docs/ssh-keys/using-ssh-keys.md (hunk 1 above)
Comment (lindseybateman) [inline content/docs/ssh-keys/using-ssh-keys.md]: check will do

File: content/docs/ssh-keys/using-ssh-keys.md (hunk 2 above)
Comment (lindseybateman) [inline content/docs/ssh-keys/using-ssh-keys.md]: check

**Pull request merged**